test/e2e/kubevirt.go

```diff
@@ -614,10 +614,14 @@ passwd:
 		}
 		vms, err := composeVMs(td.numberOfVMs, vmLabels)
 		Expect(err).ToNot(HaveOccurred())
+
 		for _, vm := range vms {
-			vm, err = kvcli.VirtualMachine(namespace).Create(context.Background(), vm)
-			Expect(err).ToNot(HaveOccurred())
+			Eventually(func() error {
+				_, err = kvcli.VirtualMachine(namespace).Create(context.Background(), vm)
```

Comment:
Can you log here each retry, In case we need to refer back to this and we need the info?

BTW, I checked the virt-api logs and while I see a bunch of GET requests being logged, I don't see any POST request logged or any other sign that the webhook was actually called? Shouldn't wee see something?

Reply:
That's the issue looks like the services is not reaching the pod, so maybe something at ovn ?

Reply:
I don't see the request logged even when it works.

Reply:
At the end looks like it's reaching kubevirt but sometimes it get stuck at io.ReadAll(req.Body) at kubevirt's webhook it could be related to fragmentatin and MTU and that could be related to github VMs.